Assets/Scripts/Collectables/HealthBasedDropper.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Actor;
+using UnityEngine;
+
+namespace Collectables
+{
+    /// <summary>
+    /// A dropper that drops drops based on the health of the entity it's bound to.
+    /// </summary>
+    public class HealthBasedDropper : Dropper
+    {
+        /// <summary>
+        /// The entity that this dropper is attached to. Used to keep track of health.
+        /// </summary>
+        public Living entity;
+        
+        /// <summary>
+        /// The health points at which to drop drops.
+        /// </summary>
+        public int[] healthMarkers;
+
+        private IList<int> sortedHealthMarkers;
```

Comment:
Continuing onward, if we go with the multi-dropper dispenser approach. Rather than an `IList` I think checking child objects and loading the amount of Dropper entities inside the object is friendly with Unity prefabs and less difficult to manage script wise. Similar to what we are doing with Boss emitters. Nice thing about this as well is you can encode location of where the drops should appear relative to the enemy (in the case of bosses). Any ideas? What do you think?

Reply:
See above response about refactor.